src/app/navigation/Home.screen.tsx

```diff
@@ -8,30 +15,35 @@ export type HomeRouteParams = undefined;
 
 export const Home = () => {
   const [movies, setMovies] = useState([]);
+  const [nbMovies, setNbMovies] = useState<number>();
 
   useEffect(() => {
-    const fetchMovies = async () => {
-      try {
-        const response = await fetch(API_URL);
-        const json = await response.json();
-        setMovies(json.results);
-      } catch (error) {
-        console.error(error);
-      }
-    };
-
-    fetchMovies();
+    fetch(API_URL)
+      .then((res) => res.json())
+      .then((json) => setMovies(json.results));
   }, []);
 
+  useEffect(() => {
+    const n = movies.length;
+    setNbMovies(n);
+  }, [movies]);
+
   const renderItem = ({ item }) => (
```

Comment:
il vaut exporter ça dans un composant à part MovieItem (keep it simple as possible)